app/controllers/questions_controller.rb

```diff
@@ -1,28 +1,44 @@
 class QuestionsController < ApplicationController
 
-  before_action :find_test, only: [:index, :create]
-  before_action :find_question, only: [:show, :destroy]
+  before_action :find_question, only: [:show, :destroy, :edit, :update]
+  before_action :find_questions, only: :index
+  before_action :find_test, only: :new
 
   rescue_from ActiveRecord::RecordNotFound, with: :rescue_with_not_found
 
   def index
-    result = @test.questions.all.pluck(:body)
-    render plain: result
+    redirect_to test_path(params[:test_id])
   end
 
-  def show
-    render plain: @question.body
-  end
+  def show; end
 
   def destroy
     @question.destroy
+    redirect_to test_questions_path(@question[:test_id])
```

Comment:
```suggestion
    redirect_to test_questions_path(@question.test)
```
лучше в объектном стиле
а с учетом, что индекс стоит вообще убрать из контроллера тут видимо поменяется редирект